roles/contiv_network/templates/aci_gw.j2

```diff
@@ -19,7 +19,7 @@ start)
     -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
     -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
     --name=contiv-aci-gw \
-    contiv/aci-gw
+    {{ aci_gw_version }}
```

Comment:
1: If someone wants to use his own custom image then just tag wont be sufficient. As we control contiv docker hub account, Unless we push his image he wont be able to try (We can do this technically by building container natively, but for someone who is not from tech background it will be difficult.)

2: Default version is mentioned in net_demo_installer script.
https://github.com/contiv/demo/pull/97